packages/cashier/src/pages/deposit/deposit.tsx

```diff
@@ -24,10 +25,10 @@ const Deposit = observer(() => {
 
     if (is_transactions_crypto_visible) return <TransactionsCryptoHistory />;
 
-    if (is_deposit || is_low_risk_cr_eu_real) {
-        if (is_crypto) return <DepositCryptoModule />;
+    if (!!currency_config && (is_deposit || is_low_risk_cr_eu_real)) {
+        const is_crypto_provider = currency_config.platform.cashier.includes('crypto');
 
-        return <DepositFiatModule />;
+        return is_crypto_provider ? <DepositCryptoModule /> : <DepositFiatModule />;
```

Comment:
Isn't it better to be specific here? ie
if `currency_config.platform.cashier.includes('crypto')` then `DepositCryptoModule`
else if `currency_config.platform.cashier.includes('doughflow')` then render `DepositFiatModule`

Reply:
Yes @wojciech-deriv 

> it gets slightly weird now as DepositFiatModule can support crypto now (if I understand the change correctly), so really, it should no longer be DepostiFiatModule, it should be "DepositDoughflowModule",

We already discussed it, unfortunately without you. And we want to change the naming to be more specific and readable. It will be done in future, since it requires refactoring.